css/main.css

```diff
@@ -301,6 +299,7 @@ tbody td {
   flex-wrap: wrap;
   justify-content: center;
   align-items: center;
+  border: 1px;
```

Comment:
@fbane Not sure on what this change achieves?

Reply:
Hmm ok. Reverted this one.